### PR TITLE
infra(add) lint-staged

### PR DESCRIPTION
**Pull request by byzyk**
/cc @montogeek

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,9 +41,12 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm test"
+      "pre-commit": "lint-staged"
     }
   },
+  "lint-staged": {
+    "*.{js,jsx,md}": ["npm test"]
```
Comment (byzyk) [inline package.json]: Although I'm not sure if we should run tests here as well. Perhaps just linting would be enough?

Comment (montogeek) [reply package.json]: What does `test` run? :D

Comment (byzyk) [reply package.json]: LOL 😄 I guess I was thinking about core repo while writing this

Comment (montogeek) [inline package.json]: We should separate this config, we already have a lint command for each file type.
It doesn't make sense to lint JS if you only updated docs.

Comment (byzyk) [reply package.json]: Sounds wise, will update today

**Comment by byzyk** [comment]
Hey @montogeek! Please have a look if it's ok to go now? I've enabled linter only for JS and MD files.

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**